ush/calcanl_gfs.py

```diff
@@ -135,7 +135,7 @@ def calcanl_gfs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix,
     ExecCMDMPI1 = ExecCMDMPI.replace("$ncmd", str(1))
     ExecCMDMPI = ExecCMDMPI.replace("$ncmd", str(nFH))
     ExecCMDLevs = ExecCMDMPI.replace("$ncmd", str(levs))
-    ExecCMDMPI10 = ExecCMDMPI.replace("$ncmd", str(10))
+    ExecCMDMPI13 = ExecCMDMPI.replace("$ncmd", str(13))
```

Comment:
Outside the scope of this PR, but it looks like this script is in need of a refactor to remove these hard-coded values that need to be changed in a half-dozen places (and to make it more pythonic in general).

Reply:
@WalterKolczynski-NOAA  Agreed!